.github/workflows/goreleaser.yml

```diff
@@ -155,19 +155,19 @@ jobs:
     runs-on: ubuntu-latest
     needs: goreleaser
     steps:
-      - name: Release file present?
+      - name: cnspec container image present?
         id: check_release_file
         uses: nick-fields/retry@v3
         with:
-          retry_wait_seconds: 10
+          retry_wait_seconds: 20
           timeout_seconds: 5
-          max_attempts: 60
+          max_attempts: 200
           retry_on: error
           # error on HTTP code different to 200
           command: |
             vSEMVER=${{ github.ref_name }}
             SEMVER="${vSEMVER//v}"
-            curl -o /dev/null -s -w "%{http_code}\n" "https://releases.mondoo.com/cnspec/${SEMVER}/cnspec_${vSEMVER}_SHA256SUMS" | grep 200          
+            curl -o /dev/null -s -w "%{http_code}\n" "https://registry.hub.docker.com/v2/repositories/mondoo/cnspec/tags/${SEMVER}-ubi-rootless" | grep 200
```

Comment:
We need the container for the installer release. The container build takes longer than the checksums file.